builder/oracle/classic/step_security.go

```diff
@@ -71,12 +151,15 @@ func (s *stepSecurity) Run(_ context.Context, state multistep.StateBag) multiste
 func (s *stepSecurity) Cleanup(state multistep.StateBag) {
 	client := state.Get("client").(*compute.ComputeClient)
 	ui := state.Get("ui").(packer.Ui)
+	config := state.Get("config").(*Config)
+
 	ui.Say("Deleting temporary rules and lists...")
 
+	namePrefix := fmt.Sprintf("/Compute-%s/%s/", config.IdentityDomain, config.Username)
```

Comment:
This seems like a somewhat common operation. Might be a good idea to add a helper fun to `config` to get this for you